src/libraries/System.Text.RegularExpressions/src/System.Text.RegularExpressions.csproj

```diff
@@ -92,6 +95,8 @@
     <Compile Include="System\Text\RegularExpressions\Symbolic\UnicodeCategoryConditions.cs" />
     <Compile Include="System\Text\RegularExpressions\Symbolic\UnicodeCategoryRanges.cs" />
     <Compile Include="System\Text\RegularExpressions\Symbolic\UnicodeCategoryRangesGenerator.cs" />
+    <Compile Include="System\Text\RegularExpressions\Symbolic\MatchReversalKind.cs"/>
+    <Compile Include="System\Text\RegularExpressions\Symbolic\MatchReversal.cs"/>
```

Comment:
Nit: we've tried to keep these lists in alphabetical order. Could you move them up to just above MintermClassifier?